grc/satellites_check_hex_string.block.yml

```diff
@@ -0,0 +1,46 @@
+# auto-generated by grc.converter
+
+id: satellites_check_hex_string
+label: Check Hex String
+category: '[Satellites]/Misc'
+
+parameters:
+-   id: hexstring
+    label: Hex String
+    dtype: string
+    default: "c0ffee"
+
+-   id: startindex
+    label: Starting Position
+    dtype: int
+    default: 0
+
+inputs:
+-   domain: message
+    id: in
+
+outputs:
+-   domain: message
+    id: ok
+    optional: true
+-   domain: message
+    id: fail
+    optional: true
+
+templates:
+    imports: import satellites
+    make: satellites.check_hex_string(${hexstring}, ${startindex})
+
+documentation: |-
+    Checks data from input port from "Starting Position" on for a "Hex String".
+
+    If the incoming string matches the entered string, data will be passed to output port "ok", else to "fail".
+
+    Example:
+    Hex String = c0ffee
+    Starting Position = 1
+
+    => Match for "ff"
```

Comment:
Can you clarify the example? It is not clear which are the parameters for the block and which is the input PDU.

Reply:
done.